FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1085,8 +1087,11 @@ extension ComposeViewController {
         }
     }
 
-    private func handleEvaluation(error: Error, with email: String) {
+    private func handleEvaluation(error: Error, with email: String, localContact: RecipientWithSortedPubKeys?) {
         let recipientState: RecipientState = {
+            if let _localContact = localContact {
+                return getRecipientState(from: _localContact)
+            }
```

Comment:
Let's call it simply `contact` instead of `_localContact`.

There's nothing wrong with the underscore but since we don't use that in other parts of the codebase, let's avoid it for consistency.

Thanks!